### ISSUE TITLE
[BUG] Anchor tags in the cookbook are without proper accessible naming

### ISSUE THREAD

**Issue by Fuzzy3** [kirbydesign/designsystem#3719]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
Currently most links (anchor tags) in the cookbook are not self-explaining - which is not a problem since they occur in a context that give them meaning. However when using a screen reader the user often traverses from link to link or even gets a complete overview of all the links shown without their surrounding context. This provides a poor experience for people using a screen reader.
This may easily be fixed by adding an aria-label to the anchor tags in the cookbook following the [accessible naming convention](https://developer.mozilla.org/en-US/docs/Glossary/Accessible_name).

## Describe how to reproduce the bug
1. Go to any page in the cookbook - could be the accessibility page: [cookbook.kirby.design/#/home/accessibility-in-kirby](https://cookbook.kirby.design/#/home/accessibility-in-kirby)
2. Turn on voiceover and make it show the links overview
**Result: The links are shown but most without proper context to understand where they will take the user.**

## What was the expected behavior?
Each link should have an [accessible name](https://developer.mozilla.org/en-US/docs/Glossary/Accessible_name) as described in the link - such that the intent of the link is clearly stated for people with disabilities.

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
